spring/spring-data/pom.xml

```diff
@@ -11,10 +11,6 @@
     <packaging>jar</packaging>
     <name>Quarkus QE TS: Spring: Spring Data</name>
     <dependencies>
-        <dependency>
-            <groupId>io.quarkus</groupId>
```

Comment:
Is that dependency was giving some problems or it's just not used?

Reply:
Resteasy Reactive does not work well with Undertow (see https://github.com/quarkusio/quarkus/issues/14997). Spite there are workarounds, this is not necessary anylonger.